src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ManageContent/ManageContentPageService.cs

```diff
@@ -1,17 +1,11 @@
 #nullable enable
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
```

Comment:
What's Rider's rationale for `System` being first when the other imports are alphabetical? [This ticket](https://youtrack.jetbrains.com/issue/RSRP-471356/Support-sorting-all-using-directives-alphabetically-including-System) references the option being added to Rider.

Reply:
I think this will have be the default option in Rider, and up until recently all team members used Rider, so a large percentage of the code will have `System` imports ordered first.

Taken from StyleCop analyzer, the rationale for ordering `System` imports first is:

>  Placing all System using directives at the top of the using directives can make the code cleaner and easier to read, and can help make it easier to identify the namespaces that are being used by the code.

I've put this as an item to discuss at the team meeting as it turns out we can try and enforce this an EditorConfig rule one way or the other which we are missing right now depending on what we decide.

See https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/dotnet-formatting-options